src/components/hero/campaignHero.tsx

```diff
@@ -17,13 +17,21 @@ const CampaignHero = () => {
 
   useEffect(() => {
     const fetchData = async () => {
-      const response = await fetch(`/api/featuredItem`);
-      const responseData = await response.json();
+      let response;
+      let responseData: FeaturedItemData;
+      try {
+        response = await fetch(`/api/featuredItem`);
+        responseData = await response.json();
+      } catch (e) {
+        console.log("error: ", e);
```

Comment:
To track the error in the console, update the message to `"CampaignHero error: ", e)`